### PR TITLE
mat: harmonise parameter naming and documentation

### LINKED ISSUE #368: mat: harmonise solve parameter naming
See https://github.com/gonum/gonum/pull/363#discussion_r159126240

### LINKED ISSUE #369: mat: harmonise solve documentation naming convention
See https://github.com/gonum/gonum/pull/363#discussion_r159126121

### PR DESCRIPTION
**Pull request by kortschak**
Please take a look.

Fixes #368.
Fixes #369.

### CODE REVIEW

**Comment by kortschak** [comment]
*ping*

**Comment by vladimir-ch** [comment]
I have the PR open in a tab (yay, that's a progress). I'll take a look later today.

File: mat/cholesky.go [hunk 1]
```diff
@@ -171,21 +171,21 @@ func (c *Cholesky) Solve(m *Dense, b Matrix) error {
...
+	blas64.Trsm(blas.Left, blas.NoTrans, 1, c.chol.mat, x.mat)
 	if c.cond > ConditionTolerance {
 		return Condition(c.cond)
 	}
 	return nil
 }
 
-// SolveChol finds the matrix m that solves A * m = B where A and B are represented
+// SolveChol finds the matrix x that solves A * X = B where A and B are represented
 // by their Cholesky decompositions a and b, placing the result in the receiver.
```
Comment (vladimir-ch) [inline mat/cholesky.go]: `...placing the result in x.`

Comment (kortschak) [reply mat/cholesky.go]: Fixed

**Pull request merged**